spec/menu_spec.rb

```diff
@@ -0,0 +1,12 @@
+require './lib/menu.rb'
+
+describe Menu do
+	it 'includes dishes with prices' do
+		expect(subject.dishes).to be_an(Array)
+		expect((subject.dishes).all? { |item| item.has_key?(:price)}).to be_truthy
```

Comment:
Why have you used "be_truthy" here? I'm not totally clear on what "truthy" does?